compiler-rt/test/asan/TestCases/Windows/allocator_may_return_null_set_via_user_function.cpp

```diff
@@ -0,0 +1,27 @@
+// RUN: %clangxx_asan -O0 %s -o %t
+// RUN: %run %t 2>&1
```

Comment:
I think you need to pipe the output through `FileCheck` for the `CHECK` comment below to do anything.

Reply:
Ah, good catch, thanks for letting me know. I was somehow under the impression that the `CHECK` pattern was implicitly always checked. I merged the tests, based on your suggestion, so that address this issue as well.